src/I18nExtension.php

```diff
@@ -32,7 +34,7 @@ public function getTokenParsers()
     public function getFilters()
     {
         return [
-            new TwigFilter('trans', 'gettext'),
+            new TwigFilter('trans', '\PhpMyAdmin\Twig\Extensions\I18nExtension::translate'), /* Note, the filter does not handle plurals */
```

Comment:
```suggestion
            new TwigFilter('trans', [$this, 'translate']), /* Note, the filter does not handle plurals */
```

Reply:
This could affect performance but I applied it and further investigation will show this